README.md

```diff
@@ -11,12 +11,55 @@ datasets with **empty raw data files**. These datasets can be useful to:
 **ALL RAW DATA FILES IN THIS REPOSITORY ARE EMPTY!**
 
 However for some of the data, the headers containing the metadata are still
-intact. This is true for the following datasets:
+intact.
+(For example the NIfTI headers for `.nii` files, or the BrainVision data
+headers for `.vhdr` files.)
+
+Headers are intact for the following datasets:
 
 - `synthetic`
 - Most EEG or iEEG data in BrainVision format (e.g., `eeg_matchingpennies`)
 
-# Validator Exceptions
+# Validating BIDS Examples
+
+The next three sections mention a few details on how the `bids-examples` can be validated using `bids-validator`.
+
+For general information on the `bids-validator`, including installation and usage, see the
+[bids-validator README file](https://github.com/bids-standard/bids-validator#quickstart).
+
+## Validating Individual Examples
+
+As mentioned above, all raw data files in this repository are empty.
+That means that for validation, the `bids-validator` needs to be configured to
+appropriately report the bids-examples as "passing"
+(see more on bids-validator configuration in the
+[bids-validator README](https://github.com/bids-standard/bids-validator#configuration)).
+
+One easy way to get around the error that the files-to-be-validated are empty,
+is by ignoring that error.
+Just run the validator as follows (using the `eeg_matchingpennies` dataset as an example,
+and assuming you are in a command line at the root of the `bids-examples` repository):
+
+`bids-validator eeg_matchingpennies --config.ignore=99`
+
+The `--config.ignore=99` "flag" will make sure that the "empty file" error will be ignored.
```

Comment:
```suggestion
The `--config.ignore=99` "flag" tells the bids-validator to ignore empty data files rather than to report the "empty file" error .
```